app/views/conversations/show.html.erb

```diff
@@ -7,50 +7,35 @@
      >
 
   <div id="left-column"
-       class="flex-col h-screen w-[260px] bg-black overflow-y-auto overflow-x-clip hidden md:flex pl-3 scrollbar-hide"
+       class="flex-col h-screen w-[260px] bg-black hidden md:flex pl-3 scrollbar-hide"
        data-click-toggle-target="destination"
    >
 
-    <div id="left-content-container" class="flex flex-col flex-grow overflow-y-auto overflow-x-clip scrollbar-hide">
-      <div id="left-content" class="flex-grow overflow-y-auto overflow-x-clip scrollbar-hide">
-        <div class="sticky top-0 pt-3 mb-1 bg-black z-10">
-          <%= render 'assistant_sidebar_link', title: "ChatGPT", settings: false %>
-        </div>
-        <%= render 'assistant_sidebar_link', title: "Samantha" %>
-        <%= render 'assistant_sidebar_link', title: "Samantha but really long so it wraps" %>
-        <%= render 'assistant_sidebar_link', title: "Samantha", selected: true %>
-        <%= render 'assistant_sidebar_link', title: "Samantha" %>
+    <div id="left-content-container" class="flex flex-col flex-grow overflow-y-auto overflow-x-clip scrollbar-hide pr-5">
+      <div class="sticky top-0 pt-3 bg-black">
+        <%= render 'assistant_sidebar_link', assistant: @conversation.assistant, settings: false, selected: true %>
```

Comment:
@krschacht right here

Reply:
If you include the currently selected assistant within the collection, then separating out this HTML becomes a little tricky. But you could move this stick-wrappy-div into the assistant_sidebar_link. When you render a collection, rails gives you a free index local variable. It'll probably be called `assistant_counter` so you can just check if it's 0 and if so render that extra div around it.

Reply:
This actually worked out nicely, but it did require updating the rails version slightly because I tripped on this bug: https://github.com/rails/rails/issues/49761

Before that merge, the `_counter` variables weren't populated when a partial had strict locals enabled.